src/DIRAC/ConfigurationSystem/Client/Helpers/Operations.py

```diff
@@ -89,63 +134,90 @@ class Operations(object):
   """ Operations class
 
       The /Operations CFG section is maintained in a cache by an Operations object
+
+      You can also inherit this class to handle another section of the configuration
+      by specifying the path to it in the mainSection argument, e.g.::
+
+        class SessionData(Operations):
+          ''' This class will have access to the configuration combined from the following sections
+              in the following order:
+                - /WebApp/
+                - /Operations/<setup>/WebApp/
+                - /Operations/<vo>/Defaults/WebApp/
+                - /Operations/<vo>/<setup>/WebApp/
+          '''
+
+          def __init__(self, credDict, setup):
+            self.__credDict = credDict
+            super(SessionData, self).__init__(group=credDict.get("group"), setup=setup, mainSection='WebApp')
+
+          def getTitle(self):
+            return self.getValue('title', 'Hello world!')
   """
 
   __cache = {}
   __cacheVersion = 0
   __cacheLock = LockRing.LockRing().getLock()
 
-  def __init__(self, vo=False, group=False, setup=False):
-    """ c'tor
+  def __init__(self, vo=None, group=None, setup=None, mainSection=''):
+    """ Determination of VO/setup and generation a list of relevant directories
 
-        Setting some defaults
+        :param str vo: VO name
+        :param str group: group name
+        :param str setup: setup name
+        :param str mainSection: main section name
+    """
+    basePath = '/Operations'
+    self._vo = getVOForGroup(group or '') or vo or getVOfromProxyGroup().get('Value', '')
+    self._setup = setup or getSetup()
+    self._group = group
+
+    # Define the configuration sections that will be merged in the following order:
+    # -> /Operations/Defaults/
+    # -> /Operations/<setup>/
+    # -> /Operations/<vo>/Defaults/
+    # -> /Operations/<vo>/<setup>/
+
+    # If mainSection defined:
+    # -> /<mainSection>/
+    # -> /Operations/<setup>/<mainSection>/
+    # -> /Operations/<vo>/Defaults/<mainSection>/
+    # -> /Operations/<vo>/<setup>/<mainSection>/
+
+    self.__paths = [os.path.join('/', mainSection)] if mainSection else [os.path.join(basePath, 'Defaults')]
+    if self._setup:
+      self.__paths.append(os.path.join(basePath, self._setup, mainSection))
+    if self._vo:
+      self.__paths.append(os.path.join(basePath, self._vo, 'Defaults', mainSection))
+      if self._setup:
+        self.__paths.append(os.path.join(basePath, self._vo, self._setup, mainSection))
+
+  def _cacheExpired(self):
```

Comment:
Is this ever used?

Reply:
I plan to use it [here](https://github.com/DIRACGrid/WebAppDIRAC/pull/498/files#diff-6e1076bea0234c7ab02f6ed366399a69eab4e8588cb5f6214e3c62a042468df0), and generally think it's a useful method